### PR TITLE
Connector21 module

### PR DESCRIPTION
**Pull request by nivancevic**

### CODE REVIEW

File: cassandra-diagnostics-connector21/pom.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        <version.bytebuddy>1.3.12</version.bytebuddy>
+        <version.aopalliance>1.0</version.aopalliance>
+        <version.cassandra>2.1.13</version.cassandra>
+        <version.plugin.compiler>3.5.1</version.plugin.compiler>
+        <version.plugin.shade>2.4.3</version.plugin.shade>
+        <version.powermock>1.6.5</version.powermock>
+        <version.spring_instrument>4.2.6.RELEASE</version.spring_instrument>
+        <version.plugin.surfire>2.19.1</version.plugin.surfire>
+        <version.hector.core>2.0-0</version.hector.core>
+        <version.cassandra.driver>3.0.0</version.cassandra.driver>
```
Comment (nbozic-zz) [inline cassandra-diagnostics-connector21/pom.xml]: It is said that 3.0 driver is compatible with 2.1 Cassandra, @mgobec it means we are fine with latest driver even though this is 2.1.x connector.

And why do we need driver, we are doing this on node level?

File: cassandra-diagnostics-connector21/pom.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        <version.junit>4.12</version.junit>
+        <version.bytebuddy>1.3.12</version.bytebuddy>
+        <version.aopalliance>1.0</version.aopalliance>
+        <version.cassandra>2.1.13</version.cassandra>
+        <version.plugin.compiler>3.5.1</version.plugin.compiler>
+        <version.plugin.shade>2.4.3</version.plugin.shade>
+        <version.powermock>1.6.5</version.powermock>
+        <version.spring_instrument>4.2.6.RELEASE</version.spring_instrument>
+        <version.plugin.surfire>2.19.1</version.plugin.surfire>
+        <version.hector.core>2.0-0</version.hector.core>
```
Comment (nbozic-zz) [inline cassandra-diagnostics-connector21/pom.xml]: We have unused dependency versions here? (powermock, hector from first glance)

File: cassandra-diagnostics-connector21/src/test/java/io/smartcat/cassandra/diagnostics/connector/ConnectorImplTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                .build()
+                .connect();
+    }
+
+    @Test
+    public void test() {
+        session.execute("SELECT * FROM system.local");
+        sleep(1000);
+        session.close();
+        Assert.assertTrue(queryCount > 0);
```
Comment (nbozic-zz) [inline cassandra-diagnostics-connector21/src/test/java/io/smartcat/cassandra/diagnostics/connector/ConnectorImplTest.java]: Lets create separate keyspace and check if we have report for that keyspace (Query class has table and keyspace and type of statement - SELECT so lets use those to verify)

File: cassandra-diagnostics-connector21/pom.xml (hunk 1 above)
Comment (nivancevic) [inline cassandra-diagnostics-connector21/pom.xml]: The driver is used only for the integration test in order to generate queries from Java.

**Pull request merged**